hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/parsing/FhirPayloadExtractor.java

```diff
@@ -18,15 +20,23 @@
     private static final JSONParser jsonParser = new JSONParser(JSONParser.DEFAULT_PERMISSIVE_MODE);
```

Comment:
The logger above (I know it's not your commit) should be static **final**.

Reply:
Done